### PR TITLE
Fixes #26051 - added grub2 pc/bios and elf options

### PR DESCRIPTION
**Pull request by lzap**
These will be useful for IBM POWER users and PC users wanting to use Grub2 on i386.

### CODE REVIEW

File: app/models/architecture.rb [hunk 1]
```diff
@@ -16,6 +16,10 @@ class Architecture < ApplicationRecord
 
   scoped_search :on => :name, :complete_value => :true
 
+  class Jail < ::Safemode::Jail
```
Comment (timogoebel) [inline app/models/architecture.rb]: Is this code related to this PR?

Comment (lzap) [reply app/models/architecture.rb]: It is intentionally not :-)

Comment (tbrisker) [reply app/models/architecture.rb]: huh? if it's not related please split it to a different PR.

Comment (lzap) [reply app/models/architecture.rb]: Resolved.

**Review by timogoebel — APPROVED** [review]

**Review by tbrisker — APPROVED** [review]
Thanks @lzap !
